client/js/components/day/day.js

```diff
@@ -18,9 +18,8 @@ var Day = React.createClass({
 
   componentWillMount: function(){
     DayStore.addChangeListener(this._handleUpdate);
-     // trigger API call to get initial data. Date.now will
-     // change once we get routing sorted
-    ViewActionCreator.getDayData(Date.now());
+     // trigger API call to get initial data.
+    ViewActionCreator.getDayData();
```

Comment:
Could you explain why we aren't passing in the date anymore to our day data fetch? Is it just because we are hardcoding the date value for now?